.github/workflows/continuous-integration.yml

```
@@ -0,0 +1,49 @@
+name: Check build pass/fail
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+
+    steps:
+      - name: Configure OS variables
+        - if: ${{ runner.os == Windows }}
```

Inconsistent indentation. It should be like this instead:
```suggestion
        - name: Configure OS variables
          if: ${{ runner.os == Windows }}
          env:
            OS: windows
```